behavior_metrics/brains/f1/rl_utils/algorithms/qlearn.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 
-
+# TODO This is a deprecated algorithm kept until we discard this way to go
```

Comment:
Can we discard this file? Why do we have this file and the following `qlearn_f1.py`?